<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StockChangeAgreementStatusEditor</class>
 <widget class="QDialog" name="StockChangeAgreementStatusEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>146</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Изменение статуса согласования</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="lblAgreementStatus">
     <property name="text">
      <string>Статус согласования</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="CDateEdit" name="edtAgreementDate">
     <property name="displayFormat">
      <string>dd.MM.yyyy</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="2" colspan="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>407</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lblAgreementNote">
     <property name="text">
      <string>Примечание к согласованию</string>
     </property>
     <property name="buddy">
      <cstring>edtAgreementNote</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>380</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblAgreement">
     <property name="text">
      <string>Дата согласования</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblAgreementPerson">
     <property name="text">
      <string>Ответственный</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QComboBox" name="cmbAgreementStatus">
     <item>
      <property name="text">
       <string>Не согласовано</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Согласовано</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Отклонено</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="1" colspan="3">
    <widget class="CPersonComboBoxEx" name="cmbAgreementPerson"/>
   </item>
   <item row="5" column="0" colspan="4">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="3">
    <widget class="QLineEdit" name="edtAgreementNote"/>
   </item>
   <item row="4" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CDateEdit</class>
   <extends>QDateEdit</extends>
   <header>library/DateEdit</header>
  </customwidget>
  <customwidget>
   <class>CPersonComboBoxEx</class>
   <extends>QComboBox</extends>
   <header>Orgs/PersonComboBoxEx.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>cmbAgreementStatus</tabstop>
  <tabstop>edtAgreementDate</tabstop>
  <tabstop>cmbAgreementPerson</tabstop>
  <tabstop>edtAgreementNote</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>StockChangeAgreementStatusEditor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>StockChangeAgreementStatusEditor</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
